packages/core/src/Camera.ts

```diff
@@ -155,6 +156,10 @@ export class Camera extends Component {
   @ignoreClone
   private _isInvViewProjDirty: BoolUpdateFlag;
   @deepClone
+  private _shaderData: ShaderData = new ShaderData(ShaderDataGroup.Camera);
```

Comment:
`_shaderData` and `_globalShaderMacro` has serious bug before？

Reply:
Done.